ui/shared/Separator.qml

```diff
@@ -2,9 +2,16 @@ import QtQuick 2.13
 import "../imports"
 
 Rectangle {
```

Comment:
I think this can be an `Item` instead, I could be wrong.

Reply:
Indeed!

Reply:
I stand corrected. There are actually several places in our app where we apply a custom `color` to `Separator`. Let's stick with `Rectangle`

Reply:
Oh, if we do apply a custom color, then this will not work as well as you think it is. It means that the color will not apply correctly plus, the height will no longer work like you implemented. Since the new color will be applied to the parent rectangle, it will be behind the children rectangle and not show, plus the custom height will be on the parent rectangle that will now be a BIG coloured separator haha.

In that case, I recommend changing to an `Item` and creating an `alias` for the `Rectangle.color`. That way, you can change the color with the same API, plus the height will still adapt correctly.

Reply:
https://github.com/status-im/status-desktop/pull/2083